sdk/java/runtime/template/pom.xml

```diff
@@ -108,6 +112,36 @@
 
         <plugins>
             <plugin>
+                <!--
+                This plugin configuration helps VS Code editor (and others) to find
+                the generated code created by `dagger init` and `dagger develop` and
+                helps for code completion.
+                -->
+                <groupId>org.codehaus.mojo</groupId>
+                <artifactId>build-helper-maven-plugin</artifactId>
+                <version>3.3.0</version>
+                <executions>
+                    <execution>
+                        <id>add-generated-sources</id>
+                        <phase>generate-sources</phase>
```

Comment:
this line is not required since `generate-sources` is the build-helper-maven-plugin default phase

Reply:
I removed it, thanks